client/src/boundary/interactor/BoardD2BConverter.ts

```diff
@@ -0,0 +1,20 @@
+import Board from "domain/Board";
+import BoundaryBoard from "boundary/model/BoundaryBoard";
+import BoundaryShip from "boundary/model/BoundaryShip";
+import { ShipD2BConverter } from "./ShipD2BConverter";
+
+export class BoardD2BConverter {
```

Comment:
įsidėk tokią klasę
`export abstract class Converter<I, O> {
    convertAll(input: I[]): O[] {
        return input.map(element => this.convert(element));
    }

    abstract convert(input: I): O;
}`
Ją extendink visuose konverteriuose, tada listams ir single objektams nereikės rašyti atskirų implementacijų